tool/dependencies/known-dependencies.txt

```diff
@@ -60,6 +60,7 @@ chill_2.12-0.7.6.jar
 classgraph-4.1.7.jar
 classmate-1.5.1.jar
 clickhouse-jdbc-0.4.6.jar
+client-3.0.0.jar
```

Comment:
Need add license and notice txt for new dependence jars
refer to  https://linkis.apache.org/zh-CN/docs/latest/development/development-specification/license